src/main/java/HW_7/Circle.java

```diff
@@ -0,0 +1,23 @@
+package HW_7;
+
+public class Circle extends Figure {
+    private double radius;
+    public static final double PI = 3.14;
```

Comment:
why the access modifier is public?

Reply:
My bad. This is constanta!!

Reply:
fixed

Reply:
not fixed

Reply:
NOT FIXED! "private static final double PI = 3.14;" should be
please revise the "constants" information